tests/playwright/src/ai-lab-extension.spec.ts

```diff
@@ -113,11 +115,26 @@ test.describe.serial(`AI Lab extension installation and verification`, () => {
       }
       await playExpect(errorTab, `Error Tab was present with stackTrace: ${stackTrace}`).not.toBeVisible();
     });
+
     test(`Verify AI Lab extension is installed`, async ({ runner, page, navigationBar }) => {
       [page, webview] = await handleWebview(runner, page, navigationBar);
       aiLabPage = new AILabPage(page, webview);
       await aiLabPage.navigationBar.waitForLoad();
     });
+
+    test(`Enable GPU support`, async ({ page, navigationBar }) => {
+      test.skip(!AI_LAB_TEST_GPU_SUPPORT, 'GPU support is not enabled');
+      test.setTimeout(120_000);
+      await aiLabPage.enableGpuSupport();
+      const settingsBar = await navigationBar.openSettings();
+      await settingsBar.expandPreferencesTab();
+      await playExpect(settingsBar.preferencesTab).toBeVisible();
+      await settingsBar.getPreferencesLinkLocator('Extension: AI Lab').click();
+      const preferencesPage = new PreferencesPage(page);
+      await playExpect(
+        preferencesPage.content.getByRole('checkbox', { name: 'Experimental GPU support for inference servers' }),
+      ).toBeChecked();
```

Comment:
This should be extracted as reusable function into ie. utilities. You should be able to verify and check the gpu in from preferences as a main function for changing the functionality rather than one time banner.